### ISSUE TITLE
Link on website to Universal Windows Platform apps is broken

### ISSUE THREAD

**Issue by rboy1** [esnet/iperf#1459]
The link on the website https://iperf.fr/iperf-download.php#uwp

is broken

https://www.microsoft.com/en-us/store/apps/network-performance-test/9nblggh5gq8f

Does not exist.

I believe it should be
https://www.microsoft.com/en-us/p/iperf/9nblggh5fgfn

Also the email on the website for sending website issues does not work

**Comment by bmah888** [comment]
https://iperf.fr/ is run by a third-party unaffiliated with ESnet. In fact we have tried to contact the maintainers of the site in order to 1) stop distributing old binaries for iperf3, 2) make it clear that they are not the developers/maintainers of iperf2 or iperf3. We have unfortunately not received any response from them in several years of trying.

**Comment by rboy1** [comment]
@bmah888 thanks for the clarification. If you don’t want them to distribute your binaries and if they aren’t responding then why do you link to their page for the Windows distribution?

https://software.es.net/iperf/obtaining.html#binary-distributions

**Comment by davidBar-On** [comment]
Following are two versions of text I prepared for myself for copy/paste about a site that distributes up to date iperf3 binaries for Windows.  Maybe this can help:

1. Newer iperf3 version for Windows is available [here](https://files.budman.pw/) (although not an official iperf3 site - maintained by [BudMan](https://www.neowin.net/forum/tags/iperf3/)).
2. There is a site that distributes new iperf3 executable for Windows. This is not an iperf3 official site but it seems to be reliable: [https://files.budman.pw/](https://files.budman.pw/) or [https://www.neowin.net/forum/topic/1234695-iperf-3101-windows-build/page/3/](https://www.neowin.net/forum/topic/1234695-iperf-3101-windows-build/page/3/) (in both download is done from the same site).

**Comment by rboy1** [comment]
I guess the important point is to link to a website where you have a relationship to manage the content on their website.

**Comment by rboy1** [comment]
@bmah888 on a related note, the windows universal platform app, is that also made by this team? Does it share the same sources?

**Comment by bmah888** [comment]
> @bmah888 on a related note, the windows universal platform app, is that also made by this team? Does it share the same sources?

@rboy1: No, we (ESnet) do not distribute binaries for any platform, and we can't vouch for the binaries made by any third-parties. That said, I _believe_ that the Windows binaries were made from unmodified sources.
